topology/probes/pmd/pmd.go

```diff
@@ -0,0 +1,122 @@
+package pmd
+
+import (
+        "time"
+        "os/exec"
+        "sync/atomic"
+        "sync"
+        "github.com/skydive-project/skydive/common"
+        "github.com/skydive-project/skydive/logging"
+        "github.com/skydive-project/skydive/topology/graph"
+)
+
+type PMDProbe struct {
+        graph.DefaultGraphListener
+        quit      chan bool
+        graph     *graph.Graph
+        state        int64
+        wg        sync.WaitGroup
+}
+func (o *PMDProbe)  OnNodeAdded(n *graph.Node) {
+        if tp, _ := n.GetFieldString("Name"); tp == "ovs-system" {
+                atomic.StoreInt64(&o.state, common.RunningState)
```

Comment:
I would call Start here, removing the need of wait group

Reply:
Thanks for the comment, will do that.
```
func (m *PMDProbe) Start() {
        m.graph.AddEventListener(m)
}
func (o *PMDProbe)  OnNodeAdded(n *graph.Node) {
                o.CoverageUpdate()
}
```